.goreleaser.yaml

```diff
@@ -12,11 +12,14 @@ builds:
       - CGO_ENABLED=0
     goos:
       - linux
+      - windows
+      - darwin
     ldflags:
       - -s -w -X main.version={{.Version}} -X main.time={{.CommitTimestamp }} -X main.buildNum={{.Version}}
 archives:
   - replacements:
       linux: Linux
+      darwin: macOS
```

Comment:
Is this replacement necessary, even for linux? I have usually seen it kept as is.

Reply:
Kept to default.